package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "body-parser": "^1.19.0",
     "cookie-parser": "^1.4.4",
-    "deltachat-node": "^1.0.0-beta.15",
+    "deltachat-node": "1.0.0-alpha.11",
```

Comment:
The master-branch has `^1.0.0-beta.15`, so that shouldn't be changed in this PR.